readme.md

```diff
@@ -104,6 +105,18 @@
 - [john](https://github.com/davej/john) - Make npm3's flat dependencies easier to find and sort.
 
 
+## Clients
+
+### Official
```

Comment:
Like this ?
***
## Table of Contents

- [Articles](#articles)
- [Tools](#tools)
- [Packages](#packages)
- [Alternatives](#alternatives)
- [Tips](#tips)
- [FAQ](#faq)
- [Community](#community)
- [Documentation](#documentation)
- [Support](#support)

## ...
## Alternatives

- [pnpm](https://github.com/rstacruz/pnpm) - Performant 'npm install'.
- [ied](https://github.com/alexanderGugel/ied) - Like npm, but faster - an alternative package manager for Node.

## ...